### DISCUSSION TITLE
Improving the Danish NER Dataset

### DISCUSSION THREAD

**Discussion by saattrupdan** [EuroEval/EuroEval#86] [General]
Currently, the Danish NER dataset used in the benchmark is [DaNE](https://huggingface.co/datasets/dane). The texts come from the [Danish Dependency Treebank](https://github.com/UniversalDependencies/UD_Danish-DDT), whose texts all come from the [Parole dataset](https://korpus.dsl.dk/resources/details/parole.html). As mentioned in [their description](https://korpus.dsl.dk/download/PAROLE.zip), these texts mainly come from newspapers:

> PAROLE-korpussets 1.553 individuelle tekstuddrag indeholder hver især mellem 140 og 180 løbende tekstord (ekskl. interpunktionstegn) og består af et eller flere efterfølgende afsnit, der er automatisk udvalgt fra DSL's elektroniske tekstbibliotek. De første ca. 100.000 løbende tekstord (643 tekstuddrag) er udvalgt fra Den Danske Ordbogs (DDO) SGML-opmarkerede tekstkorpus på 40 mio. løbende tekstord bestående af uddrag af dagblade, bøger, tidsskrifter osv5. De sidste ca. 150.000 løbende tekstord (910 tekstuddrag) er udvalgt fra DSL's avistekstbibliotek og er blevet SGML-opmarkeret til PAROLE-korpusset. De fleste tekster i dette korpus er således uddrag af avistekster (jf. appendiks 8.4 om korpusteksternes klassificering ifølge medium).

However, a new Danish NER has appeared: [DANSK](https://huggingface.co/datasets/chcaa/DANSK). The texts in this corpus are sampled from the [Danish Gigaword Corpus](https://gigaword.dk/), with diverse domains. The [NER scheme in the DANSK dataset](https://huggingface.co/datasets/chcaa/DANSK#dataset-composition) is different from the standard CoNLL scheme, and would need to be mapped if we want to use it.

**Comment by KennethEnevoldsen** [comment]
I believe this relies on #85.

**Reply by saattrupdan** [reply]
Yeah agreed.
